uniprot-config/src/main/resources/return-fields-config/uniparc-crossref-return-fields.json

```diff
@@ -0,0 +1,179 @@
+[
+  {
+    "seqNumber": 0,
+    "itemType": "GROUP",
+    "groupName": "Names & Taxonomy",
+    "isDatabaseGroup": false,
+    "id": "names_&_taxonomy"
+  },
+  {
+    "parentId": "names_&_taxonomy",
+    "childNumber": 0,
+    "itemType": "SINGLE",
+    "isRequiredForJson": true,
+    "isDefaultForTsv": true,
+    "defaultForTsvOrder" : 0,
+    "label": "Database",
+    "name": "database",
+    "paths": ["database"],
+    "id": "names_&_taxonomy/database",
+    "includeInSwagger": true
+  },
+  {
+    "parentId": "names_&_taxonomy",
+    "childNumber": 1,
+    "itemType": "SINGLE",
+    "isRequiredForJson": true,
+    "isDefaultForTsv": true,
+    "defaultForTsvOrder" : 1,
+    "label": "Identifier",
+    "name": "accession",
+    "paths": ["id"],
+    "id": "names_&_taxonomy/accession",
+    "includeInSwagger": true
+  },
+  {
+    "parentId": "names_&_taxonomy",
```

Comment:
Can you please verify  "isDefaultForTsv" and "defaultForTsvOrder" property for all the fields, because in current uniprot website there are 13 fields as default. and in this config file there are just 7. Can you please add the missing ones and make sure that the order is correct as well.

Reply:
I verified again, there are 7 default fields for TSV.